playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -43,17 +43,23 @@
     append: yes
 
 - name: Ensure kubectl is installed
-  include: kubectl-install.yml
+  include_role:
```

Comment:
I like the idea here. But the airship roles are the ones used for the product deployment. I'd recommend go the other direction to have deploy_osh to include the airship role instead.

Reply:
In general I would not care, but there's slight difference in one of the file where we've already added some change in osh role but not in airship

Reply:
I don't care which version to keep either. If the osh version has more features than the airship version, we can move the osh version over to the airship role folder.